gpkit/nomials/variables.py

```diff
@@ -160,3 +181,13 @@ def __new__(cls, shape, *args, **descr):
         obj.key = VarKey(**obj.descr)
 
         return obj
+
+
+class VectorizableVariable(Variable):
```

Comment:
inheritance is tricky here, should it double-inherit from `Variable` and `ArrayVariable`?